src/components/Uploader/style.css

```diff
@@ -52,3 +52,11 @@
     margin-top: 4.5rem;
   }
 }
+
+.clientLgScreen {
```

Comment:
where in the code is it used?

Reply:
This is used in the cross device client and it handles cases in which a user opens the mobile form on a large screen (ie desktop, ipad or landscape mobile).

Reply:
I would recommend renaming it following the convention that css style is part of a selector that indicates semantically which component is being styled.
either with inheritance `.crossDeviceClient .dropzone` or `dropzoneCrossDeviceClient`